src/components/SideBar.tsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react'
+import { makeStyles } from '@material-ui/core/styles';
+import {
+    Container,
+    Typography,
+    Box,
+    Button,
+    Grid
+} from '@material-ui/core'
+
+const useStyles = makeStyles({
+    root: {
+        marginTop: '4.5rem',
+        borderStyle: 'solid',
+        textAlign: 'center',
+        marginLeft: '-2rem'
+    },
+
+    root2: {
+        padding: '1rem',
+    },
+
+    searchBy: {
+        marginTop: '1.5rem',
+    },
+
+    filters: {
+        background: 'lightgrey',
+        marginTop: '1rem'
+    }
+
+});
+
+
+const SideBar: React.FC = () => {
+    const classes = useStyles();
+
+    return (
+
+        <Grid container className={classes.root}>
+            <Box className={classes.root2}>
+                <Grid item>
+                    <Typography variant="h5">
+                        Welcome to the C4C Jumpstart Blog!
+                        Feel free to checkout the recent posts
+                        and contribute to our post collection.
+                    </Typography>
+                </Grid>
+                <Grid item className={classes.searchBy}>
+                    <Typography>Search By:</Typography>
+                    <Button className={classes.filters}>Likes</Button> <br />
```

Comment:
Let's avoid `<br />`

Reply:
Also try to keep each component on its own line. So if we're going to keep that `<br />`, put it on the line below.

Reply:
fixed it